Useful tools & utilities/Libraries.md

```diff
@@ -1,28 +1,22 @@
 ## Views
 
-1. **Butterknife**
-
-	Butterknife is a view "injection" library that reduces boilerplate code which you have to write to cast the corresponding views in your layout. This library is a must for all of our projects.
-
-	More info with code examples is available [here](http://jakewharton.github.io/butterknife/).
-
-2. **CircleImageView**
+1. **CircleImageView**
 
 	A fast circular ImageView perfect for profile images. As this is just a custom ImageView and not a custom 	Drawable or a combination of both, it can be used with all kinds of drawables, e.g. a PicassoDrawable from Picasso or other non-standard drawables.
 
 	More info with code examples is available [here](https://github.com/hdodenhof/CircleImageView).
 
-3. **Holo Graph Library**
+2. **Glide**
 
-	Slick, easy-to-use graphs with click listeners.
+	Glide is a fast and efficient open source media management and image loading framework for Android. It wraps media decoding, memory and disk caching, and resource pooling into a simple and easy-to-use interface. It also has support for animated GIFs.
 
-	More info with code examples is available [here](https://github.com/Androguide/HoloGraphLibrary).
+	More info with code examples is available [here](https://github.com/bumptech/glide).
 
-4. **Glide**
+3. **MPAndroidChart**
 
-	Glide is a fast and efficient open source media management and image loading framework for Android. It wraps media decoding, memory and disk caching, and resource pooling into a simple and easy-to-use interface. It also has support for animated GIFs.
+    MPAndroidChart is a powerful graph view library supporting line, pie, bar radar, bubble graphs and much more. It also includes other features like scaling, panning and animations.
 
-	More info with code examples is available [here](https://github.com/bumptech/glide).
+    More info with code examples is available [here](https://github.com/PhilJay/MPAndroidChart).
 
 
 ## Networking
```

Comment:
I recommend we remove GSON from this list, as we're not using it in any new projects, and just stick with Moshi 🙂

Reply:
Sure!